docs/source/contributing/documentation.rst

```diff
@@ -382,12 +386,103 @@ When adding type hints to manim, there are some guidelines that should be follow
 
 *  numpy arrays can get type hints with ``np.ndarray``
 
-Missing Sections
-----------------
+* assuming that typical path objects are either Paths or strs, one can use the typehint ``typing.Union[str, pathlib.Path]``
+
+Missing Sections for typehints are:
 * Tools for typehinting
 * Link to MyPy
 * Mypy and numpy import errors: https://realpython.com/python-type-checking/#running-mypy
 * Where to find the alias
 * When to use Object and when to use "Object".
 * The use of a TypeVar on the type hints for copy().
 * The definition and use of Protocols (like Sized, or Sequence, or Iterable...)
+
+
+Adding Blocks for Tip, Note, Important etc. (Admonitions)
+---------------------------------------------------------
+
+
+The following directives are called Admonitions. You
+can use them to point out additional or important
+information.
+
+Examples
+========
```

Comment:
Make sure to use the proper section. This doesn't render in the docs because it's using the same `====` section heading as the top level:
> ====================
Adding Documentation
====================